tests/test_runners/test_playwright.py

```diff
@@ -10,9 +10,20 @@ class PlaywrightTest(CliTestCase):
     @responses.activate
     @mock.patch.dict(os.environ,
                      {"LAUNCHABLE_TOKEN": CliTestCase.launchable_token})
-    def test_record_test_rspec(self):
+    def test_record_test(self):
         result = self.cli('record', 'tests', '--session', self.session,
                           'playwright', str(self.test_files_dir.joinpath("report.xml")))
 
         self.assert_success(result)
         self.assert_record_tests_payload('record_test_result.json')
+
+    @responses.activate
+    @mock.patch.dict(os.environ,
+                     {"LAUNCHABLE_TOKEN": CliTestCase.launchable_token})
+    def test_record_test_with_json_option(self):
```

Comment:
Could you please add the failed test case in tests/data/playwright/report.json?

Reply:
I wanted to do that and tried it... However, on the Windows environment, some characters didn't match correctly
I gave up  and ignored error case

Reply:
Oh, I see.

Reply:
Can we do something like this: https://github.com/launchableinc/cli/blob/main/tests/test_testpath.py#L94-L99?

Reply:
Can we skip Windows? If ok, I'll update it

Reply:
It should work.

https://docs.python.org/3/library/unittest.html#unittest.skipIf